packages/variable-resolver/src/browser/variable-resolver-service.ts

```diff
@@ -150,6 +150,10 @@ export namespace VariableResolverService {
             return true;
         }
 
+        isInteractionsNOK(): boolean {
+            return this.options.interactionsState ? this.options.interactionsState.isNOK() : false;
+        }
```

Comment:
Can be simplified to:

```suggestion
        isInteractionsNOK(): boolean {
            return this.options.interactionsState?.isNOK() ?? false;
        }
```

or 

```suggestion
        isInteractionsNOK(): boolean {
            return !!this.options.interactionsState?.isNOK()
        }
```